pkg/api/v1/schema_test.go

```diff
@@ -114,14 +117,14 @@ var exampleJSON = `{
             "name": "disk0",
             "volumeName": "volume0",
             "disk": {
-              "dev": "vda"
+              "bus": "virtio"
```

Comment:
Are you sure that virtio was the default before?

Reply:
yes, albeit implicitely since we were expecting "vd*" device name.